internal/charmstore/store.go

```diff
@@ -719,40 +723,92 @@ var seriesScore = map[string]int{
 	"utopic":  4,
 	"vivid":   5,
 	"wily":    6,
+	// When we find a multi-series charm (no series) we
+	// will always choose it in preference to a series-specific
+	// charm
+	"": 5000,
```

Comment:
Given that we've got a known set of integers enumerated here, maxint seems like overkill to me.